package.json

```diff
@@ -66,6 +66,7 @@
     "react-switch": "^5.0.1",
     "react-use": "^17.2.3",
     "react-webcam": "^5.2.0",
+    "react-window": "^1.8.6",
```

Comment:
Added library solves the issue at the expense of some inline styles used.

Not super ergonomic, so it may add a bit of debt that can be repaid by searching for another.

Combined with `@types/react-window` adds under 900kB https://packagephobia.com/result?p=react-window@1.8.6

Reply:
https://reactjs.org/docs/optimizing-performance.html#virtualize-long-lists lists the following:

- https://react-window.now.sh/
- https://bvaughn.github.io/react-virtualized/

Reply:
https://github.com/bvaughn/react-virtualized#a-word-about-react-window

> If you're considering adding react-virtualized to a project, take a look at react-window as a possible lighter-weight alternative. Learn more about how the two libraries compare here.

https://github.com/bvaughn/react-window#how-is-react-window-different-from-react-virtualized

Reply:
This also sounds interesting:

- https://github.com/bvaughn/react-window-infinite-loader

Reply:
Also looks like an interesting read: https://www.smashingmagazine.com/2020/03/infinite-scroll-lazy-image-loading-react/

Reply:
I went through all of those links before I settled to just using `react-window` out of 2 resons:

- They build on top of older versions  or the current version of `react-window` to tailor for specific cases, nothing game changing
- Their interface depends on the same inline styles used by `react-window`

So, I opted for the minimal and encapsulated way I could use `react-window` in case might find another or roll out our own that takes into account the `.scss` styles e.g. by inspecting the DOM element itself and/or preset `var(--iables)`

Reply:
All that related to the library, https://www.smashingmagazine.com/2020/03/infinite-scroll-lazy-image-loading-react/ is in line to what I was considering might replace `react-window` as a more robust solution